src/endpoints/cart.js

```diff
@@ -82,7 +82,7 @@ class CartEndpoint extends BaseExtend {
   BulkAdd(body) {
     return this.request.send(`${this.endpoint}/${this.cartId}/items`, 'POST', {
       data: body,
-      options: { add_all_or_nothing: false }
+      options: { add_all_or_nothing: true }
```

Comment:
hey @krish4uu thanks for opening this

we agree the current implementation of this isn't the best. however we have some concerns about making the change this way. this could be breaking behaviour for any downstream implementation that relies on this being `false`

that's not necessarily a problem because we can release breaking changes safely if necessary, but a better implementation might be to allow the user to specify this value instead of defaulting it either way. perhaps this method could take an optional `options` param allowing the user to specify an options object to be included on the request?

it would also be great if we could update the [types](https://github.com/moltin/js-sdk/blob/main/src/types/cart.d.ts#L305) and [tests](https://github.com/moltin/js-sdk/blob/main/test/unit/cart.ts#L703) to match

we are happy to help out if needed 👍

Reply:
Hi @samblacklock, sure I can try. Please let me know whom should I contact for help, if needed, thank you

Reply:
@krish4uu sure just post in in our [slack channel](https://elasticpath-software.slack.com/archives/C03Q69DC9U4)